uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/request-modal/details/leave.html

```diff
@@ -5,13 +5,13 @@
       <div class="col-xs-8">
         <div class="btn-group btn-group-justified">
           <label
-            ng-disabled="$ctrl.isMode('view') || $ctrl.isRole('manager')"
+            ng-disabled="$ctrl.isMode('view') || ($ctrl.isRole('manager') && !$ctrl.isLeaveType('sickness'))"
```

Comment:
According to current condition, all the items will be disabled for manager, if the leave type is not `sickness`. But the everything should be enabled for a manager regardless of the leave type?